lib/package.json

```
@@ -0,0 +1,72 @@
+{
+    "name": "@asgardeo/token-exchange-plugin",
+    "version": "0.0.1",
```

`0.1.0` is the suggested initial semantic version.

```suggestion
    "version": "0.1.0",
```